### PR TITLE
docs: add flutter user management video to flutter user management guide

### PR DESCRIPTION
**Pull request by dshukertjr**
## What kind of change does this PR introduce?

Adds a video embedding of the flutter user management video on the Flutter user management guide page.

### CODE REVIEW

**Comment by dshukertjr** [comment]
@thorwebdev @dijonmusters I noticed that the Next.js and Remix user management guide doesn't have the videos embedded, but is there a particular reason for it?

File: apps/docs/pages/guides/getting-started/tutorials/with-flutter.mdx [hunk 1]
```diff
@@ -5,6 +5,15 @@ export const meta = {
   description: 'Learn how to use Supabase in your Flutter App.',
```
Comment (gregnr) [inline apps/docs/pages/guides/getting-started/tutorials/with-flutter.mdx:5]: @dshukertjr we're starting to move videos to the TOC section for better screen real estate:
image

which can be done using the `tocVideo` key in `meta`:
```suggestion
  description: 'Learn how to use Supabase in your Flutter App.',
  tocVideo: 'r7ysVtZ5Row',
```

File: apps/docs/pages/guides/getting-started/tutorials/with-flutter.mdx [hunk 2]
```diff
@@ -5,6 +5,15 @@ export const meta = {
...
 
+<div className="video-container">
+  <iframe
+    src="https://www.youtube-nocookie.com/embed/r7ysVtZ5Row"
+    frameBorder="1"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+    allowFullScreen
+  ></iframe>
+</div>
+
```
Comment (gregnr) [inline apps/docs/pages/guides/getting-started/tutorials/with-flutter.mdx:16]: Then we no longer would need this here:

```suggestion
```

**Review by gregnr — APPROVED** [review]

File: apps/docs/pages/guides/getting-started/tutorials/with-flutter.mdx (hunk 2 above)
Comment (dshukertjr) [reply apps/docs/pages/guides/getting-started/tutorials/with-flutter.mdx:16]: Thanks @gregnr ! I wasn't aware of that! That looks fantastic!

**Pull request merged**